### PR TITLE
feat: add licenses

### PR DESCRIPTION
**Pull request by bashybaranaba**

### CODE REVIEW

File: public/licenses/png/exlusive-noncommercial-color.png
Comment (eddiemg10) [inline public/licenses/png/exlusive-noncommercial-color.png:1]: I see you updated the non-commercial, Nice! 
Looks really good

File: public/licenses/png/arttribute-exclusive-license.png
Comment (eddiemg10) [inline public/licenses/png/arttribute-exclusive-license.png:1]: For the Arttribute License, I noticed our font's "L" looks like an I so looks more like AI than AL. Idk if we should stick with the lowercase L

Comment (bashybaranaba) [reply public/licenses/png/arttribute-exclusive-license.png:1]: I was also debating this in my head, coz AL looking like AI kinda speaks to the point of these licenses being for Gen AI purposes😅. Would love to hear more of your thoughts on this.

Comment (eddiemg10) [reply public/licenses/png/arttribute-exclusive-license.png:1]: Haha didn't think of that. Honestly don't have too strong of an opinion on this. Maybe bring it up in the next meeting to see what the others will say if you hadn't already

**Pull request merged**